scripts/tests/traits/dual_wield.lua

```diff
@@ -0,0 +1,97 @@
+-----------------------------------
+-- Dual Wield Trait - Equipment TP Reset Prevention
+-- This test verifies that TP is preserved when equipment changes
+-- fail due to missing Dual Wield trait.
+--
+-- Note: Dual Wield is available on:
+--   - NIN (Ninja) - Level 10
+--   - DNC (Dancer) - Level 20
+--
+-- Fixes: https://github.com/HorizonFFXI/HorizonXI-Issues/issues/2268
+-----------------------------------
+
+describe('Dual Wield Trait - Equipment TP Reset Prevention', function()
+    ---@type CClientEntityPair
+    local player
+
+    before_each(function()
+        -- Spawn NIN/5 (before DW unlocks at level 10)
```

Comment:
But comments like this are fine, because it isn't obvious that this happens from `xi.test.world:spawnPlayer...`

Reply:
Don't like "NIN/5" when all players would know it would be "NIN5" or "level 5 ninja"